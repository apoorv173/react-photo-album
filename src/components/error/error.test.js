import React from "react";
import Error from "./error";
import { shallow, mount } from "enzyme";

describe("Error component Component", () => {

    const onClose = jest.fn();

    it("Should render the error message", () => {
        const tree = shallow(<Error />)
        expect(tree).toMatchSnapshot();
    });

    it('should render all the components isnide the error component and its heading', () => {
        const wrapper = mount(<Error error={'No data found'}/>);
        expect(wrapper.find('h1').length).toBe(1);
    });

});