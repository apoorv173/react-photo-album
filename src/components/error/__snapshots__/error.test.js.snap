// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component Component Should render the error message 1`] = `ShallowWrapper {}`;
